src/basis/entity.js

```diff
@@ -856,12 +816,15 @@
 
         args.push(name, fname);
         values.push(defValue, fields[key]);
+
+        var newValueArgument =
+          'has.call(data,' + escapedKey + ')' +
+            '?' + 'data[' + escapedKey + ']' +
+            ':' + name + (typeof defValue == 'function' ? '(data)' : '');
+        var oldValueArgument = defValue !== undefined && typeof defValue !== 'function' ? ',' + name : '';
```

Comment:
Obviously, yes, we do. This change needed in order to support passing initial default value to transforms. Previously transforms such as `dateField` or `arrayField` in case of incorrect transformation falled back to `oldValue || null`. But basis.type does not contain such logic. Anyway, this needed for all such transforms, not only array or date